fcrepo-configs/src/main/resources/config/file-s3/repository.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name" : "repo",
+    "jndiName" : "",
+    "workspaces" : {
+        "predefined" : ["default"],
+        "default" : "default",
+        "allowCreation" : true,
+        "cacheSize" : 10000
+    },
+    "storage" : {
+        "persistence": {
+            "type": "file",
+            "path" : "${fcrepo.object.directory:target/objects}"
+        },
+        "binaryStorage" : {
+            "type" : "s3",
+            "username" : "${aws.accessKeyId}",
```

Comment:
I have concerns about adding support for `JAVA_OPTS="${JAVA_OPTS} -Daws.secretKey=..."`

This effectively encourages Fedora users to put their AWS secret key in the JAVA_OPTS configuration, which can be retrieved by anyone with shell access to the machine:

`ps -ax | grep aws.secretKey`

While this is certainly convenient, I'm not sure that it is a usage pattern that should be encouraged, let alone supported.

These same concerns apply to the use of database passwords.